client/packages/requisitions/src/ResponseRequisition/DetailView/ResponseLineEdit/ResponseLineEdit.tsx

```diff
@@ -30,31 +31,50 @@ export const ResponseLineEdit = ({
     useDraftRequisitionLine(currentLine);
   const { next, hasNext } = useNextResponseLine(currentLine);
   const { data } = useResponse.line.stats(draft?.id);
+  const {
+    numberOfPacksToQuantity,
```

Comment:
I think it's ok to useUnitVariant again in ResponseLineEditForm, don't really need to pass it on as props

Reply:
Thought it better to pass as props since it's used in `ResponseStoreStats`, `RequestStoreStats` and `ResponseLineEditForm` but can change if you think it's better to call it again in those places (: